tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -281,7 +294,9 @@ def parse_value_per_device(devices, values_string, value_type):
 def process_help_inference_string(benchmark_app, device_number_streams):
     output_string = f'Start inference {benchmark_app.api_type}hronously'
     if benchmark_app.api_type == 'async':
-        output_string += f', {benchmark_app.nireq} inference requests'
+        output_string += ', ' if output_string else ''
```

Comment:
from my perspective this `if-else` doesn't make sense as `output_string` always isn't empty